src/System.Memory/tests/Performance/Perf.Utf8Formatter.cs

```diff
@@ -0,0 +1,83 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text;
+using System.Runtime.CompilerServices;
+using Microsoft.Xunit.Performance;
+using Xunit;
+
+namespace System.Buffers.Text.Tests
+{
+    public static partial class Utf8FormatterTests
+    {
+        private const int InnerCount = 100000;
+
+        // There are really only two integer formatters: Int64/UInt64. Benchmarking the others won't provide any extra code coverage.
+
+        [Benchmark(InnerIterationCount = InnerCount)]
+        [InlineData(12837467L)] // standard format
+        [InlineData(-9223372036854775808L)] // min value
+        [InlineData(9223372036854775807L)] // max value
+        private static void ParserInt64(long value)
```

Comment:
How do the results compare to UTF16 routines in mscorlib?

Reply:
Improvement

  44%  // 12837467L
 -26%  // long.MinValue
  13%  // long.MaxValue

("long.MinValue" has a special path just for it in FormatInt64D() that delegates to the unsigned formatter - looks like this is an outlier case. 'Course, if we really think people will benchmark that a lot, it is easy to write a super-fast path when it's only for one specific value...)